From: David Paleino <dapal@debian.org>
Subject: fix paths to point to correct locations
Forwarded: not-needed

---
 ninja_ide/resources.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- ninja-ide.orig/ninja_ide/resources.py
+++ ninja-ide/ninja_ide/resources.py
@@ -58,7 +58,7 @@ LANGS_DOWNLOAD = os.path.join(HOME_NINJA
 
 EDITOR_SKINS = os.path.join(HOME_NINJA_PATH, "addins", "schemes")
 
-START_PAGE_URL = os.path.join(PRJ_PATH, "doc", "startPage.html")
+START_PAGE_URL = "/usr/share/doc/ninja-ide/startPage.html"
 
 NINJA_THEME = os.path.join(PRJ_PATH, "addins", "theme", "ninja_dark.qss")
 
